.eslintrc.js

```diff
@@ -0,0 +1,225 @@
+        "constructor-super": "error",
+        "curly": "off",
+        "default-case": "error",
+        "eol-last": "error",
+        "eqeqeq": [
+            "error",
+            "smart"
+        ],
+        "func-style": [
+          "error",
+          "declaration",
+           { "allowArrowFunctions": true }
+        ],
+        "guard-for-in": "off",
+        "id-blacklist": [
+            "error",
+            "any",
+            "Number",
+            "number",
+            "String",
+            "string",
+            "Boolean",
+            "boolean",
+            "Undefined",
+            "undefined"
+        ],
+        "id-match": "error",
+        "jsdoc/check-alignment": "error",
+        "jsdoc/check-indentation": "error",
+        "jsdoc/newline-after-description": "error",
+        "max-len": [
+            "error",
+            {
+                "code": 100
+            }
+        ],
+        "no-bitwise": "error",
+        "no-caller": "error",
+        "no-cond-assign": "error",
+        "no-console": [
+            "error",
+            {
+                "allow": [
+                    "log",
+                    "warn",
+                    "error",
+                    "dir",
+                    "assert",
+                    "clear",
+                    "count",
+                    "countReset",
+                    "group",
+                    "groupCollapsed",
+                    "groupEnd",
+                    "Console",
+                    "dirxml",
+                    "table",
+                    "markTimeline",
+                    "profile",
+                    "profileEnd",
+                    "timeline",
+                    "timelineEnd",
+                    "timeStamp",
+                    "context"
+                ]
+            }
+        ],
+        "no-control-regex": "error",
+        "no-debugger": "error",
+        "no-empty": "error",
+        "no-extra-semi": "error",
+        "no-eval": "error",
+        "no-fallthrough": "error",
+        "no-invalid-regexp": "error",
+        "no-irregular-whitespace": "error",
+        "no-multiple-empty-lines": "error",
+        "no-new-wrappers": "error",
+        "no-null/no-null": "off",
+        "no-redeclare": "error",
+        "no-regex-spaces": "error",
+        "no-shadow": [
+            "off",
+            {
+                "hoist": "all"
+            }
+        ],
+        "no-throw-literal": "error",
+        "no-trailing-spaces": "error",
+        "no-unused-labels": "error",
+        "no-var": "error",
+        "object-curly-spacing": [
+            "error",
+            "always"
+        ],
+        "prefer-const": "error",
+        "radix": "error",
+        "spaced-comment": [
+            "error",
+            "always",
+            {
+                "markers": [
+                    "/"
+                ]
+            }
+        ],
+        "use-isnan": "error",
+        /*
+        "@typescript-eslint/tslint/config": [
```

Comment:
This is a list of rules from TSLint that were not automatically copied over to ESLint.

I think the biggest one is probably `function-name`, which specifies the format for private/static/function methods (like `__private_method`). I don't know how to make that happen in ESLint.

Reply:
There is no replacement yet. I think we're ok to go without this rule.